src/briefcase/commands/create.py

```diff
@@ -652,8 +673,21 @@ def install_app_requirements(self, app: AppConfig, test_mode: bool):
 
         try:
             requirements_path = self.app_requirements_path(app)
-            self._write_requirements_file(app, requires, requirements_path)
         except KeyError:
+            requirements_path = None
+
+        try:
+            requirement_installer_args_path = self.app_requirement_installer_args_path(
+                app
+            )
+        except KeyError:
+            requirement_installer_args_path = None
+
+        if requirements_path:
+            self._write_requirements_file(
+                app, requires, requirements_path, requirement_installer_args_path
+            )
+        else:
```

Comment:
This change in logic is to isolate the indication of whether the requirements files should be written to only if `app_requirements_path` is defined. To ensure backwards compatibility, `app_requirement_installer_args_path` needs to be optional. Existing template versions won't define it but should continue to write the requirements path.